tide_core.install

```diff
@@ -1446,3 +1446,48 @@ function tide_core_update_8061() {
   $field_storage->setSetting('allowed_values', $allowed_values);
   $field_storage->save();
 }
+
+/**
+ * Changes Linkit profile.
+ */
+function tide_core_update_8062() {
+  // Update allowed HTML list.
+  $config_factory = \Drupal::configFactory();
+  $editor_ids = [
+    'editor.editor.admin_text',
+    'editor.editor.rich_text',
+    'editor.editor.summary_text',
+  ];
+  foreach ($editor_ids as $editor_id) {
+    $editor = $config_factory->getEditable($editor_id);
+    if ($editor->isNew()) {
+      continue;
+    }
+    $editor->set('settings.plugins.drupallink.linkit_profile', 'site_content');
+    $editor->save();
+  }
+}
+
+/**
+ * Changes to filter weights.
+ */
+function tide_core_update_8063() {
+  // Update allowed HTML list.
+  $config_factory = \Drupal::configFactory();
+  $filter_ids = [
+    'filter.format.admin_text',
+    'filter.format.rich_text',
+    'filter.format.summary_text',
+  ];
+  foreach ($filter_ids as $filter_id) {
+    $filter = $config_factory->getEditable($filter_id);
+    if ($filter->isNew()) {
+      continue;
+    }
+
+    $filter->set('filters.filter_align.weight', '-50');
+    $filter->set('filters.filter_caption.weight', '-49');
+    $filter->set('filters.linkit.weight', '-48');
+    $filter->save();
```

Comment:
Hi @Jeannie-deloitte , I can only see the above update hook only updating the weights but not the `allowed HTML list`. Unless I am missing something. 
I can see [allowed_html](https://github.com/dpc-sdp/tide_core/pull/331/files#diff-bfaac244bb002238402fbeca2505a522242ef93e3a9415e21ace38579cbb725dR51), https://github.com/dpc-sdp/tide_core/pull/331/files#diff-bfaac244bb002238402fbeca2505a522242ef93e3a9415e21ace38579cbb725dR99 was updated but not added in the update hook. So shouldn't those be also added to the update hook

Reply:
@MdNadimHossain Thanks for pointing that out. I didnt actually change the allowed HTML list (line 1475 is an accidental comment left in from copying another update hook). The config file upon export from Drupal has rearranged the Allowed HTML list however the actual list hasnt changed. I will update the comment to be less confusing